addons/web_editor/static/src/js/editor/image_processing.js

```diff
@@ -213,10 +215,17 @@ async function applyModifications(img, dataOptions = {}) {
         originalSrc,
         glFilter,
         filterOptions,
+        cropEnabled,
```

Comment:
Maybe this should be named differently: it makes me think that the image won't be cropped if false while it only concerns GIF, right?